gitlab-ce/inventory/group_vars/all.yml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
Better approach is to create a file called `seed-hosts.yml` to avoid dumping everything into `all.yml`. Mind making that change?